### PR TITLE
Improve linking between baseline and working copy

### PR DESCRIPTION
**Pull request by Nachtalb**
- Add a link from working copy to baseline
- Instead of showing the full URL of the working copy on the baseline, show an a tag with linking to the working copy

DE working copy ==> baseline
image

EN baseline ==> working copy
image

### CODE REVIEW

**Comment by jone** [comment]
@Nachtalb tests are failing here

File: ftw/simplelayout/locales/de/LC_MESSAGES/ftw.simplelayout.po [hunk 1]
```diff
@@ -263,9 +263,9 @@ msgstr "Deinstalliert das ftw.simplelayout.mapblocks Paket"
 msgid "VideoBlock"
 msgstr "Video"
 
-#: ./ftw/simplelayout/contenttypes/contents/videoblock.py:43
-msgid "Youtube format: http(s)://youtu.be/VIDEO_ID<br/>Vimeo format: http(s)://vimeo.com/(channels/groups)/VIDEO_ID"
-msgstr "Youtube Format: http(s)://youtu.be/VIDEO_ID<br/>Youtube (no-cookie) Format: https://www.youtube-nocookie.com/embed/VIDEO_ID<br/>Vimeo Format: http(s)://vimeo.com/(channels/groups)/VIDEO_ID"
+#: ./ftw/simplelayout/contenttypes/contents/videoblock.py:50
+msgid "Youtube format: http(s)://youtu.be/VIDEO_ID<br/>Youtube (no-cookie) format: https://www.youtube-nocookie.com/embed/VIDEO_ID<br/>Vimeo format: http(s)://vimeo.com/(channels/groups)/VIDEO_ID"
+msgstr ""
```
[the remark this answers is not in the document]
Comment (Nachtalb) [reply ftw/simplelayout/locales/de/LC_MESSAGES/ftw.simplelayout.po]: No, I just ran `i18n-build` and committed the changes in the locals. But maybe I accidentally deleted this. I'll check it out.

[the remark this answers is not in the document]
Comment (Nachtalb) [reply ftw/simplelayout/locales/de/LC_MESSAGES/ftw.simplelayout.po]: Ahh The `i18n-build` updated the `msgid` and removed the `msgstrs` it is the same.

[the remark this answers is not in the document]
Comment (jone) [reply ftw/simplelayout/locales/de/LC_MESSAGES/ftw.simplelayout.po]: If the msgid has changed, it is a new message, thus empty. Can you add the missing message string here?

**Comment by Nachtalb** [comment]
@jone I readded the translation

**Pull request merged**